yarn-project/circuits.js/src/tests/factories.ts

```diff
@@ -430,12 +434,12 @@ export function makePrivateKernelInputsInner(seed = 1): PrivateKernelInputsInner
  * @param seed - The seed to use for generating the public call stack item.
  * @returns A public call stack item.
  */
-export function makePublicCallStackItem(seed = 1): PublicCallStackItem {
+export function makePublicCallStackItem(seed = 1, full = false): PublicCallStackItem {
   const callStackItem = new PublicCallStackItem(
     makeAztecAddress(seed),
     // in the public kernel, function can't be a constructor or private
     new FunctionData(makeSelector(seed + 0x1), false, false),
-    makePublicCircuitPublicInputs(seed + 0x10),
+    makePublicCircuitPublicInputs(seed + 0x10, undefined, full),
```

Comment:
I **think** you can skip the undefined by just using named parameters like so:
`makePublicCircuitPublicInputs(seed + 0x10, full=full)`